api-operator/pkg/controller/api/api_controller.go

```diff
@@ -407,12 +434,15 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 
 			var kanikoJob *batchv1.Job
 			log.Info("Deploying the Kaniko job in cluster")
+			r.recorder.Event(instance, corev1.EventTypeNormal, "KanikoJob", "Deploying kaniko job.")
 			kanikoJob = kaniko.Job(instance, controlConfigData, kanikoArgs.Data[kanikoArguments], ownerRef)
 			if err := controllerutil.SetControllerReference(instance, kanikoJob, r.scheme); err != nil {
 				return reconcile.Result{}, err
 			}
 			// create Kaniko job and set kaniko object
 			if errJob := k8s.CreateIfNotExists(&r.client, kanikoJob); errJob != nil {
+				r.recorder.Event(instance, eventTypeError, "KanikoJob",
+					"Error when creating the kaniko job.")
```

Comment:
same here

Reply:
Those have been removed in the latest code.